### DISCUSSION TITLE
可以推荐一款 vue 的 图片剪裁组件吗?

### DISCUSSION THREAD

**Discussion by zqm971210** [zxuqian/frontend-questions#7] [求助]
## 可以推荐一款 vue 的 图片剪裁组件吗?
我在百度上查了查 没有找到太好的 o(╥﹏╥)o

**Comment by zxuqian** [comment]
这个不好说，Github 上可以搜索出好多来，符合你的需求就行

**Comment by EGo14T** [comment]
https://github.com/xyxiao001/vue-cropper
